src/locales/sv.json

```diff
@@ -756,4 +751,4 @@
       "linkText": "SV: Avaa rekisteriseloste tästä (pdf)"
     }
   }
-}
+}
```

Comment:
A line break should be added to end of file.

Reply:
Line break added.